test/house_test.rb

```diff
@@ -0,0 +1,135 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/room'
+require './lib/house'
+
+class HouseTest < Minitest::Test
+  def test_a_house_exists
+    house = House.new("$400000", "123 sugar lane")
+    assert_instance_of House, house
+  end
+
+  def test_house_has_a_price
+    house = House.new("$400000", "123 sugar lane")
+    assert_equal 400000, house.price
+  end
+
+  def test_house_has_an_address
+    house = House.new("$400000", "123 sugar lane")
+    assert_equal "123 sugar lane", house.address
+  end
+
+  def test_house_starts_with_no_rooms
+    house = House.new("$400000", "123 sugar lane")
+    assert_equal [], house.rooms
+  end
+
+  def test_that_you_can_add_rooms
+    house = House.new("$400000", "123 sugar lane")
+    room_1 = Room.new(:bedroom, 10, '13')
+    room_2 = Room.new(:bedroom, 11, '15')
+    house.add_room(room_1)
+    house.add_room(room_2)
+    assert_equal 2, house.rooms.count
```

Comment:
Try to be as specific as possible with your test assertions.  With this assertion, the following method would pass, but is not doing what is intended:

```ruby
def add_room(room)
  @rooms << "I'm adding a room, now"
end
```

You want to make sure that the elements in the rooms array is what you expect:
```ruby
assert_equal [room_1, room_2]. house.rooms
```